src/sections/General/Navigation/index.js

```diff
@@ -207,11 +221,14 @@ const Navigation = ({ theme, themeSetter }) => {
   return (
 
     <ThemeProvider theme={theme === "dark" ? darktheme : lighttheme}>
-      <NavigationWrap className={`nav-block ${scroll ? "scrolled" : ""}`}>
+      <NavigationWrap className={`nav-block ${scroll ? "scrolled" : ""}`} ref={navWrapRef}>
         <Container className="nav-container">
           <div className="navbar-wrap">
             <Link to="/" className="logo">
-              <img src={theme === "dark" ? layer5dark_logo : layer5_logo} alt="Layer5 logo" />
+              <img src={theme === "dark" ? layer5dark_logo : layer5_logo} alt="Layer5 logo"
+                onClick={function () {
+                  setExpand(false); closeDropDown();
+                }}/>
```

Comment:
Let's not add it to the logo, just having an event listener for outside the header is fine.

Reply:
Okay! Thanks for the reply. I will make the said changes.